laurent22/joplin
# Mobile,Desktop: Upgrade KaTeX to v0.16.10

# Summary

Updates KaTeX to version 0.16.10.

# Testing

Relevant tests include:
https://github.com/laurent22/joplin/blob/7ee5cad21e0e2f0ee652eb22ae61177f178f421f/packages/app-desktop/integration-tests/main.spec.ts#L43
https://github.com/laurent22/joplin/blob/7ee5cad21e0e2f0ee652eb22ae61177f178f421f/packages/app-cli/tests/MdToHtml.ts#L300

Manual testing:
1. Create a new note with content
    ```
    $$\texttt{Block TeX}$$
    
    Inline $\KaTeX$.
    
    ```
2. Select all, copy, paste, paste.
3. Repeat step 2 nine more times.
4. Verify that the note renders successfully.

This has been tested successfully on Ubuntu 24.04. Similar testing was done on Android 14, however, repeating step 2 fewer times.
